### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by kavanagh**
This doesn't work properly yet. There's a problem with eslint and I cant figure they problem.

### CODE REVIEW

**Comment by kavanagh** [comment]
@callumlocke Are you able to figure this out? I sure as hell cant.

To reproduce: pull this branch, run `npm install` and let Atom try do it's eslinting.

**Comment by callumlocke** [comment]
@kavanagh @aendrew please check this over.

As discussed:

- To lint locally, you can run the standard `eslint .` command to use airbnb's linting (or just use your editor's lint highlighter, which essentially does the same thing)
- In CI, we run `npm run lint` which does `eslint --no-eslintrc --config=.eslintrc-ci.yml .`
  - This uses the new [eslint-config-ft-graphics](https://github.com/ft-interactive/eslint-config-ft-graphics), which is a place we can add rules we deem important enough to break the build.

In summary: we needed two different linting strategies, one for editor highlighting (to gently encourage better style) and one for CI (to ban the deployment of downright bugs). The cleanest way to do this is to have two separate linting configs, rather than dynamically generating a multipurpose frankenstein config that's clever and unmaintainable and deviates from eslint norms.
